darts/tests/utils/historical_forecasts/test_historical_forecasts.py

```diff
@@ -112,6 +123,8 @@
 
     NB_EPOCH = 1
 
+    models += [NLinearModel, RNNModel]
```

Comment:
one torch model should be enough

```suggestion
    models += [NLinearModel]
```